Lib/test/audit-tests.py

```diff
@@ -567,6 +567,15 @@ def hook(event, args):
     _winapi.CreateNamedPipe(pipe_name, _winapi.PIPE_ACCESS_DUPLEX, 8, 2, 0, 0, 0, 0)
 
 
+def test_assert_unicode():
+    import sys
+    sys.addaudithook(lambda *args: None)
+    try:
+        sys.audit(9)
+    except TypeError:
+        pass
```

Comment:
Should make it so this test fails if no exception is raised, e.g. with an `else` block.

Reply:
Thank you very much, added an `else` that raises `RuntimeError`.